doc/callbacks/file_writing.md

```diff
@@ -0,0 +1,6 @@
+# File writing callbacks
```

Comment:
I feel this documentation could be expanded a little bit.

e.g. give a full example of how to use this callback including setting the directory to put the files into and controlling which columns to write out?